src/ga4gh/core/_internal/identifiers.py

```diff
@@ -210,6 +210,10 @@ def identify_all(
     if input_obj is None:
         return None
     output_obj = input_obj
+    if isinstance(input_obj, str):
+        if input_obj.startswith("ga4gh:") and not input_obj.startswith("ga4gh:SQ"):
```

Comment:
I'm not sure I understand this. Why do we not strip the `ga4gh:SQ.` to get the digest from sequence object identifiers?